src/content/docs/dropins/product-details/pdp-functions.mdx

```diff
@@ -21,12 +21,12 @@ getProductData(sku: string);
 
 ## getRefinedProduct
 
-A function that returns refined product's data. It takes `sku` and `optionUIDs` as parameters.
+A function that returns refined product's data. It takes `sku`, `optionUIDs`, and `isBundle?` as parameters.
 
 ```js
 import { getRefinedProduct } from '@/pdp/api/getRefinedProduct';
 
-getRefinedProduct(sku: string, optionUIDs: string[]);
+getRefinedProduct(sku: string, optionUIDs: string, isBundle?: boolean[]);
 ```
 
```

Comment:
ℹ️ I think that the _initialData_ might be missing from the docs (it will be similar to the cart one from https://experienceleague.adobe.com/developer/commerce/storefront/dropins/cart/cart-containers/#initialdata). We can look on improving the docs in a separate PR. But just to explain, this is how the initialData is being passed:
https://git.corp.adobe.com/magento-datalake/storefront-pdp/blob/develop/examples/html-host/index.html#L191-L194
and it gets defined like this:
https://git.corp.adobe.com/magento-datalake/storefront-pdp/blob/develop/examples/html-host/index.html#L165-L168.

Kind of unrelated, sorry, but I don't us to forget about this.

Reply:
@alandana,

Could it be that we need to change the name of `ProductData` to `initialData` [here](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/product-details/pdp-containers/#productdata)?

Reply:
No, it isn't really the same thing. The `initialData` applies through the `initializers`, as it can be seen in the Cart docs.
I'll make a note to see if/how `productData` is used.